geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -192,7 +192,7 @@ Could not connect to Redis at 127.0.0.1:6379: Connection refused
 | MSETNX | PERSIST | PEXPIRE | PEXPIREAT |
 | PING | PSETEX | PSUBSCRIBE | PTTL |
 | PUBLISH PUBSUB | PUNSUBSCRIBE | RENAME |
-| RENAMENX | RESTORE | SADD | SCARD |
+| RENAMENX | RESTORE | RPOP | SADD | SCARD |
```

Comment:
This will break the formatting of the command list table, which currently has 4 columns, as this line now has 5. It's a massive pain, but currently in order to add a new command, you need to move every following command too. I've suggested on other PRs that it would make life easier to break this table up into multiple tables for each command type (Sets, Lists, etc.) as that would mean less editing is needed each time a new command is added.